meta-mender-core/classes/mender-setup.bbclass

```diff
@@ -176,6 +176,67 @@ python() {
         bb.fatal("MENDER_PARTITION_ALIGNMENT_MB is deprecated. Please define MENDER_PARTITION_ALIGNMENT_KB instead.")
 }
 
+addhandler mender_vars_handler
+python mender_vars_handler() {
+    if(isinstance(e, bb.event.ParseCompleted)):
+        from bb import data
+        import os
+        import re 
+        import json
+
+        path = d.getVar("LAYERDIR_MENDER")
+        path = os.path.join(path, "conf/mender-vars.json")
+
+        if( (os.path.isfile(path))):
+            mender_vars = {}
+            with open(path, "r") as f:
```

Comment:
if not using the with scope, why not make this a one-liner? (:

Reply:
Isn't he using it...?

Reply:
It's no biggie, but I was thinking, why not simply:
```
 mender_vars = json.load(open(path, "r"))
```

Reply:
with statement implicitly calls f.close()

Reply:
haha, kk. I give